resources/data.py

```diff
@@ -249,14 +252,35 @@ def generate(rm: ResourceManager):
         rm.item_tag('forge:gems', 'tfc:gem/' + gem)
 
     for wood in WOODS.keys():
-        rm.item_tag('minecraft:logs', 'tfc:wood/log/%s' % wood, 'tfc:wood/wood/%s' % wood, 'tfc:wood/stripped_log/%s' % wood, 'tfc:wood/stripped_wood/%s' % wood)
         rm.item_tag('twigs', 'tfc:wood/twig/%s' % wood)
         rm.item_tag('lumber', 'tfc:wood/lumber/%s' % wood)
         rm.item_tag('sluices', 'tfc:wood/sluice/%s' % wood)
-        rm.item_tag('looms', 'tfc:wood/planks/%s_loom' % wood)
+        rm.item_tag('looms', 'tfc:wood/planks/%s_loom' % wood)        
+        rm.item_tag('minecraft:boats', 'tfc:wood/boat/%s' % wood)
+        rm.item_tag('minecraft:planks', 'tfc:wood/planks/%s' % wood)
+        rm.item_tag('minecraft:wooden_buttons', 'tfc:wood/planks/%s_button' % wood)
+        rm.item_tag('minecraft:wooden_fences', 'tfc:wood/planks/%s_fence' % wood)
+        rm.item_tag('minecraft:wooden_fences', 'tfc:wood/planks/%s_log_fence' % wood)
+        rm.item_tag('forge:fence_gates/wooden', 'tfc:wood/planks/%s_fence_gate' % wood)
+        rm.item_tag('minecraft:wooden_slabs', 'tfc:wood/planks/%s_slab' % wood)
+        rm.item_tag('minecraft:wooden_stairs', 'tfc:wood/planks/%s_stairs' % wood)
+        rm.item_tag('forge:bookshelves', 'tfc:wood/planks/%s_bookshelf' % wood)
+        rm.item_tag('minecraft:wooden_doors', 'tfc:wood/planks/%s_door' % wood)
+        rm.item_tag('minecraft:wooden_trapdoors', 'tfc:wood/planks/%s_trapdoor' % wood)
+        rm.item_tag('minecraft:wooden_pressure_plates', 'tfc:wood/planks/%s_pressure_plate' % wood)
+        rm.item_tag('forge:crafting_table', 'tfc:wood/planks/%s_workbench' % wood)
+        rm.item_tag('forge:lectern', 'tfc:wood/lectern/%s' % wood)
+        rm.item_tag('minecraft:leaves', 'tfc:wood/leaves/%s' % wood)
+
         if wood in TANNIN_WOOD_TYPES:
             rm.item_tag('makes_tannin', 'tfc:wood/log/%s' % wood, 'tfc:wood/wood/%s' % wood)
 
+        if wood in VANILLA_WOODS:
+            rm.item_tag('minecraft:%s_logs' % wood, 'tfc:wood/log/%s' % wood, 'tfc:wood/wood/%s' % wood, 'tfc:wood/stripped_log/%s' % wood, 'tfc:wood/stripped_wood/%s' % wood)
+        else:
+            rm.item_tag('forge:%s_logs' % wood, 'tfc:wood/log/%s' % wood, 'tfc:wood/wood/%s' % wood, 'tfc:wood/stripped_log/%s' % wood, 'tfc:wood/stripped_wood/%s' % wood)
+            rm.item_tag('minecraft:logs', '#forge:%s_logs' % wood)
```

Comment:
I don't really like this at all. For consistency reasons we should be leaving these all in one namespace. Namely, the vanilla namespace. Packmakers are not gonna think to switch back and forth based on what vanilla has